package.json

```diff
@@ -94,7 +91,20 @@
       "@semantic-release/git",
       "@semantic-release/github"
     ],
-    "branch": "master"
+    "branches": [
+      "+([0-9])?(.{+([0-9]),x}).x",
+      "main",
+      "next",
```

Comment:
Not sure we want to trigger a release on next or next-major

Reply:
Hopefully this was answered by @JamesSingleton  we are overriding semantic release method to ensure that we can use main branch. This are the inbuilt functions